### PR TITLE
Remove suppression: remove the unused local variable 'x' (#664)

### PR DESCRIPTION
**Pull request by sindydwns**
resolved
- #664 
I removes the sonar analyzer suppression (remove unused local variable 'x') and resolves build errors that result from it.

### CODE REVIEW

**Review by OnedgeLee — APPROVED** [review]

**Review by OnedgeLee — APPROVED** [review]

File: Libplanet.Net/BlockCompletion.cs [hunk 1]
```diff
@@ -438,7 +438,7 @@ public async Task WaitAll(CancellationToken cancellationToken = default)
                     if (tasks.Any())
                     {
                         var tcs = new TaskCompletionSource<object>();
-                        using CancellationTokenRegistration ctr = cancellationToken.Register(
+                        cancellationToken.Register(
```
Comment (longfin) [inline Libplanet.Net/BlockCompletion.cs:441]: Declaring `ctr` (even no usage) seemed be intended due to auto disposing. 🤔 (see also: https://github.com/SonarSource/sonar-dotnet/issues/3137)

so I think we may need to keep previous with new comments to suppress lint.
cc @planetarium/libplanet

Comment (sindydwns) [reply Libplanet.Net/BlockCompletion.cs:441]: ok thanks. i will check it!

Comment (dahlia) [reply Libplanet.Net/BlockCompletion.cs:441]: As Sonar Analyzer does not provide any way to discard this pattern of usage yet, we should manually suppress the warning at this moment, like:

```csharp
#pragma warning disable S1481  // Though var _ is unused, the using statement is still needed
using CancellationTokenRegistration _ = cancellationToken.Register(
    () => tcs.TrySetCanceled(),
    useSynchronizationContext: false
);
#pragma warning restore S1481
```

**Review by dahlia — CHANGES_REQUESTED** [review]
Could you update *Libplanet.ruleset* and *Libplanet.Tests.ruleset* as well?

Also, you need to add `[changelog skip]` to your commit message.

**Pull request closed without merging**